test/manydbs/manydbs.c

```diff
@@ -186,7 +187,7 @@ main(int argc, char *argv[])
         }
     }
 
-    sleep(10);
+    sleep(30);
```

Comment:
Did you mean to extend the test duration? I'm guess that is what this controls?

Reply:
Oops was debugging! placed back